deployment/deploy/deploy.sh

```diff
@@ -104,13 +104,46 @@ else
     UPDATE_NGINX=1
 fi
 
+filterConf() {
+    SRC="$1"
+    DST="$2"
+    SERVER_NAME="$3"
+    PROXY_REDIRECT="$4"
+    INTERNAL_PORT="$5"
+
+    if [ ! -f "$DST" ]; then
+        sed -e "s/\${DOMAIN_NAME}/${DOMAIN_NAME}/" "$SRC" | \
+        sed -e "s/\${SERVER_NAME}/${SERVER_NAME}/" | \
+        sed -e "s/\${PROXY_REDIRECT}/${PROXY_REDIRECT}/" | \
+        sed -e "s/\${INTERNAL_PORT}/${INTERNAL_PORT}/" >"$DST"
+    fi
+}
+
+deploySite() {
+    NAME="$1"
+    SERVER_NAME="$2"
+    PROXY_REDIRECT="$2"
+    INTERNAL_PORT="$3"
+
+    echo "Deploying site $NAME"
+    filterConf nginx/site-template "/etc/nginx/sites-available/$NAME" $SERVER_NAME $PROXY_REDIRECT $INTERNAL_PORT
```

Comment:
I know server name should not have spaces nor should a proxy URL or port - but I would suggest to add quotes anyway...

Reply:
And done